tracing-flame/Cargo.toml

```
@@ -0,0 +1,13 @@
+[package]
+name = "tracing-flame"
+version = "0.1.0"
+authors = ["Jane Lusby <jlusby@yaah.dev>"]
+edition = "2018"
+
+# See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
+
+[dependencies]
+tracing-subscriber = { version = "0.2.3", default-features = false, features = ["registry", "fmt", "env-filter"] }
+tracing = { version = "0.1.13" }
```

nit: if we're not going to require any features or other configurations, this can just be 
```suggestion
tracing  = "0.1.13"
```